src/storage/storage.ts

```diff
@@ -72,6 +72,21 @@ export class Storage implements IStorage {
     this.version = version;
   }
 
+  /**
+   * Return the deposit percentage for creating tokens.
+   * @returns {number}
+   */
+  getTokenDepositPercentage(): number {
+    if (this.version && this.version.token_deposit_percentage) {
+      return this.version.token_deposit_percentage;
+    }
+    /**
+     *  When using wallet-service facade we do not update the version constants
+     *  Since this data is important for the wallets UI we will return the default value here.
+     */
+    return TOKEN_DEPOSIT_PERCENTAGE;
```

Comment:
If the `token_deposit_percentage` is `0`, should we return `0` or the default?

Also, could this be simplified?

```suggestion
    return this.version?.token_deposit_percentage ?? TOKEN_DEPOSIT_PERCENTAGE
```

(either `??` or `||` should be used, depending on the first question)

Reply:
`token_deposit_percentage` is returned from the fullnode api, it won't be 0 on our networks but I think a private network could have 0 for it.